openaerostruct/tests/test_aerostruct_wingbox_wave_fuel_vol_constraint_opt.py

```diff
@@ -289,7 +289,7 @@ def test(self):
             # Airfoil properties for viscous drag calculation
             "k_lam": 0.05,  # percentage of chord with laminar
             # flow, used for viscous drag
-            "t_over_c_cp": np.array([0.08, 0.08, 0.08, 0.10, 0.10, 0.08]),
+            "t_over_c_cp": 0.12 * np.ones(6),  # np.array([0.08, 0.08, 0.08, 0.10, 0.10, 0.12]),

```

Comment:
What’s this change for?

Reply:
Oh I see now in your comment. Could you remove the commented out code then? Otherwise this LGTM

Reply:
0h damn yeah forgot to remove that